src/sedpack/io/dataset_iteration.py

```diff
@@ -121,28 +120,28 @@ def shard_paths_dataset(
         return shard_paths
 
     def read_and_decode(self, tf_dataset: TFDatasetT,
```

Comment:
```suggestion
    def read_and_decode(self, tf_dataset: TFDatasetT, cycle_length: int | None,
```

Reply:
Yapf multiline output is a bit hard to parse in GitHub workflows (I did not find the time to do it yet). It just wants to move `cycle_length` to this line. And GitHub code suggestions for multiline are also strange (or I should go to sleep).